### PR TITLE
html-relative-links - Yuliia P.

### PR DESCRIPTION
**Pull request by yuliia-p**
https://user-images.githubusercontent.com/106217355/176248836-78d91385-1829-466a-9731-fdd7fb4f12fc.mp4

### CODE REVIEW

File: html-relative-links/about-me.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <li>About Me</li>
+    <li>
+      <a href="projects.html">Projects</a>
+    </li>
+    <li>
+      <a href="contact-us.html">Contact Us</a>
+    </li>
+  </ul>
+  <p>Hello About Me!</p>
+  <img src="https://pbs.twimg.com/media/EwxVoP5W8Ac4cVV.jpg" alt="Girl holding programming book">
```
Comment (codingcodymiller) [inline html-relative-links/about-me.html:27]: You should be using the images in the `images` directory

File: html-relative-links/about-me.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  </style>
+</head>
+
+<body>
+  <ul>
+    <li>
+      <a href="index.html">Home Page</a>
+    </li>
+    <li>About Me</li>
```
Comment (codingcodymiller) [inline html-relative-links/about-me.html:18]: Missed this one.

File: html-relative-links/about-me/about-me.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <ul>
+    <li>Home Page</li>
+    <li>About Me</li>
+    <li>Projects</li>
+    <li>Contact Us</li>
+  </ul>
+  <p>Hello About Me!</p>
+  <img src="???" alt="Girl holding programming book">
```
Comment (codingcodymiller) [inline html-relative-links/about-me/about-me.html:19]: None of this is complete.

File: html-relative-links/contact-us/contact-us.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+</head>
+<body>
+  <ul>
+    <li>Home Page</li>
+    <li>About Me</li>
+    <li>Projects</li>
+    <li>Contact Us</li>
+  </ul>
+  <p>Hello Contact Us!</p>
+  <img src="???" alt="Girl holding programming book">
```
Comment (codingcodymiller) [inline html-relative-links/contact-us/contact-us.html:19]: None of this is complete.

File: html-relative-links/contact-us.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+<body>
+  <ul>
+    <li>
+      <a href="index.html">Home Page</a>
+    <li>
+      <a href="about-me.html">About Me</a>
+    <li>
+      <a href="projects.html">Projects</a>
+    </li>
+    <li>Contact Us</li>
```
Comment (codingcodymiller) [inline html-relative-links/contact-us.html:22]: Missed this one.

File: html-relative-links/contact-us.html [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  </ul>
+  <p>Hello Contact Us!</p>
+  <img
+    src="https://images.squarespace-cdn.com/content/v1/60b7f1bed70b3159a0b9141d/1645987750387-5MXLHRAKAIAM78AUZKYI/photo6109534113649242172.jpg"
```
Comment (codingcodymiller) [inline html-relative-links/contact-us.html:26]: Again, use the images in the `images` directory

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Make sure to double-check that you have completed everything requested before submitting for review 👀

**Comment by yuliia-p** [comment]
https://user-images.githubusercontent.com/106217355/176264794-207e8ad3-48c5-43d7-b144-54132b4f2e94.mp4

**Comment by yuliia-p** [comment]
https://user-images.githubusercontent.com/106217355/176343869-a45914e6-8ffc-4c85-9062-2a0509a90105.mp4

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**